src/include/daos_fs.h

```diff
@@ -1,5 +1,5 @@
 /*
- * (C) Copyright 2018-2021 Intel Corporation.
```

Comment:
I just did a copyright update patch so if you merge latest master, that should cut down on these.